### PR TITLE
Content tweak on chap2, 3

### PR DESCRIPTION
**Pull request by jkrvivian**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -57,19 +57,17 @@ \section{\normalsize\textbf{Related Work}}
...
 
-Such work proposed different solutions to specific issues of data marketplaces. In this paper, we propose an overall design of decentralized data marketplace that handles data integrity and trading procedures such as buyout and subscriptions to future data on a distributed ledger.
-
 \section{\normalsize\textbf{System Architecture}}
-Our proposed data marketplace framework is a 3-tier decentralized architecture with a registrar who is responsible for marketplace registration in order to post participants' information on distributed ledgers for validation, data providers who publish data, data consumers who search for interested products and issue a new trade, and brokers who interact between data providers and consumers, including data publishing, product metadata generation and trading process.
+Our proposed data marketplace framework is a 3-tier de-centralized architecture with registrars, data providers, data consumers and brokers.
```
Comment (jserv) [inline paper.tex]: Remove trailing spaces.

Comment (jserv) [reply paper.tex]: Shall we use "de-centralized" or "decentralized "?

File: paper.tex [hunk 2]
```diff
@@ -86,23 +84,19 @@ \subsubsection{Broker}
...
 
 \subsubsection{IPFS}
-Inter-Planetary File system(IPFS)\cite{IPFS} is a peer-to-peer network for storing and accessing files, websites, applications, and data in a distributed file system which is not maintained with certain nodes or entities but is with all IPFS users. In our proposed architecture, brokers are responsible to publish the metadata of products, including title, data provider information and data preview to the IPFS in order to provide users with search capabilities to meet the consumers' need.
+Inter-Planetary File system(IPFS)\cite{IPFS} is a peer-to-peer network for storing and accessing files, websites, applications, and data in a distributed file system which is not maintained with certain nodes or entities but all IPFS users. In our proposed architecture, brokers are responsible for publishing metadata to IPFS, including titles, data provider information and data preview, in order to provide users with searching capabilities.
```
Comment (jserv) [inline paper.tex]: Replace "in order to" with "to".

**Comment by jserv** [comment]
Fix conflicts.

File: paper.tex (hunk 1 above)
Comment (jkrvivian) [reply paper.tex]: decentralized!

**Pull request merged**